### DISCUSSION TITLE
no copilot chat icon

### DISCUSSION THREAD

**Discussion by titikaka0723** [community/community#55596] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![2651684203532_ pic](https://github.com/community/community/assets/130803364/cdfa0ca7-2b34-4139-9680-4d83670deeef)
![2661684203690_ pic](https://github.com/community/community/assets/130803364/2e24e3f8-81b8-4807-b3e4-3e1257a89dfd)
I have restart the editor and authorize GitHub Copilot many times, but I still cannot see chat icon.

**Comment by adrnbttr** [comment]
I have the same issue if someone has a solution...

**Comment by davecheney** [comment]
Please double check you have installed the latest build of vscode-insiders, and that you keep up to date with both the -insiders build, and the copilot-nightly extension as they change daily.

**Reply by adrnbttr** [reply, marked as answer]
Effectively, I was on the stable version of VSCode and not on the insider's version. My bad...

**Comment by GerardoTurin** [comment]
Do I need the vscode-insiders version to be able to see the GitHub Copilot Chat icon?

**Reply by davecheney** [reply]
Yes, vscode-insiders and copilot-nightly are required at the moment.
